modules/nf-core/sentieon/Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM debian:bullseye-20220328-slim AS downloader
```

Comment:
what happened to the old dockerfile?
I gues conda is still broken?

Reply:
I have no idea

Reply:
@maxulysse : Don't you wanna build the docker-image and publish it on 

https://hub.docker.com/r/nfcore/sentieon

before we merge this?

Reply:
the image is already built and pushed to quay.io